src/benchmark_register.cc

```diff
@@ -114,7 +114,14 @@ bool BenchmarkFamilies::FindBenchmarks(
   // Make regular expression out of command-line flag
   std::string error_msg;
   Regex re;
-  if (!re.Init(spec, &error_msg)) {
+  std::string prefix("-");
```

Comment:
is this used?

Reply:
Ah not anymore, this was left over from some test code, will remove :)